lib/index.js

```diff
@@ -3,8 +3,11 @@ module.exports = {
     toTree: require('./toTree'),
     pathsComplementFromTree: require('./pathsComplementFromTree'),
     pathsComplementFromLengthTree: require('./pathsComplementFromLengthTree'),
+    toJsonKey: require('./toJsonKey'),
+    isJsonKey: require('./toJsonKey').isJsonKey,
     hasIntersection: require('./hasIntersection'),
     toPaths: require('./toPaths'),
+    isSafeNumber: require('./toPaths').isSafeNumber,
```

Comment:
Maybe name this `isIntegerKey` for API consistency?

Reply:
yup, done